packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -38,40 +39,114 @@ export class AuthenticatorService {
 
     const authService = interpret(machine, {
       devTools: process.env.NODE_ENV === 'development',
-    })
-      .onTransition((state) => {
-        this._user = state.context.user;
-        this._authState = state;
-      })
-      .start();
-
-    this._services = getSendEventAliases(authService.send);
+    }).start();
+
+    this._subscription = authService.subscribe((state) => {
+      this._authState = state;
+      this._facade = getServiceContextFacade(state);
+    });
+
+    this._sendEventAliases = getSendEventAliases(authService.send);
     this._authService = authService;
   }
 
-  public get services() {
-    return this._services;
+  ngOnDestroy(): void {
+    if (this._subscription) this._subscription.unsubscribe();
+  }
+
+  /**
+   * Context facades
+   */
+
+  public get error() {
+    return this._facade?.error;
+  }
+
+  public get hasValidationErrors() {
+    return this._facade?.hasValidationErrors;
+  }
+
+  public get isPending() {
+    return this._facade?.isPending;
+  }
+
+  public get route() {
+    return this._facade?.route;
+  }
+
+  public get user() {
+    return this._facade?.user;
   }
 
+  public get validationErrors() {
+    return this._facade?.validationErrors;
+  }
+
+  /**
+   * Service facades
+   */
+
+  public get updateForm() {
+    return this._sendEventAliases.updateForm;
+  }
```

Comment:
I used getters here, or else `this` will get lost when it goes through `ngTemplateOutlet`. This is resolvable by using the good old `.bind(this)` on the constructor, but imo this is cleaner.